app/models/admin.rb

```diff
@@ -3,5 +3,23 @@ class Admin < ApplicationRecord
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable,
-         :recoverable, :rememberable, :trackable, :validatable
+         :recoverable, :rememberable, :trackable
+
+  def email_required?
+    true
+  end
```

Comment:
This method looks very useless, it will never return false. Maybe it needs to have some logic?